---
title: "Exploring marine genomic observation data"
output: rmarkdown::html_vignette
vignette: >
  %\VignetteIndexEntry{lwdata_genetic}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
---

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>",
  warning = FALSE,
  message = FALSE
)
```

<img src='lwde_genetic.png' class='rightimg'/>

Genomic observations in the Belgian Part of the North Sea.

- Partners: [Flanders Marine Institute (VLIZ)](https://www.vliz.be/en/imis?module=institute&insid=36)
- Period: since June 2012
- Geographical coverage: [Belgian Exclusive Economic Zone](https://www.marineregions.org/gazetteer.php?p=details&id=3293) and [Southern Bight of the North Sea](https://marineregions.org/gazetteer.php?p=details&id=2399)
- Taxonomic coverage: Several Bacteria and Eukaryota phylums
- Moratorium: None
- Data quality: Validated
 
## How to interpret this dataset
 
### Query options
* **Timeframe**: Starting and stopping date for the query.

### Available columns

* **Project**: Project in charge of sampling.
* **Station**: Name or code of sampling station.
* **TripAction**:	Unique identifier to recognize actions taken aboard the RV Simon Stevin.
* **ActionType**: The type of sampling equipment.
* **OTUId**: Unique identifier for each Operational Taxonomic Unit (OTU)
* **Sequences**: Amount of sequences assigned to Operational Taxonomic Unit (OTU).
* **MarkerGene**: Marker gene that was used for barcoding.
* **Time**: Time in UTC, beginning of sample timeperiod.
* **Longitude**: Center longitude in decimal degrees, WGS84.
* **Latitude**: Center latitude in decimal degrees, WGS84.
* **Domain**:	Taxonomic group Domain.
* **SuperPhylum**:	Taxonomic group SuperPhylum.
* **Phylum**:	Taxonomic group Phylum.
* **Class**:	Taxonomic group Class.
* **Order**:	Taxonomic group Order.
* **Family**:	Taxonomic group Family.
* **Genus**: Taxonomic group Genus.
* **Species**: Taxonomic group of organisms or group of artefacts.	
* **AphiaID**: Unique and persistent identifier of the taxon in [marinespecies.org](https://www.marinespecies.org/) (WoRMS).

## Abstract

```{r abstract, results = 'asis', echo = FALSE}
# Get the abstract and/or description from IMIS and print here
library(jsonlite)

dasid <- fromJSON("https://www.vliz.be/en/imis?module=dataset&dasid=5188&show=json")

abstract <- dasid[["datasetrec"]][["EngAbstract"]]
description <- dasid[["datasetrec"]][["EngDescr"]]

if (is.null(abstract) & is.null(description)){
  
} else if(is.null(abstract) & !is.null(description)){
  out <- description
} else if(!is.null(abstract) & is.null(description)){
  out <- abstract
} else if(!is.null(abstract) & !is.null(description)){
  out <- paste0(abstract, "</br>", description)
}

cat(out)
```

## How to cite these data

```{r get citation and license, results = 'asis', echo = FALSE}
library(jsonlite)

# Set base url
url <- "https://www.vliz.be/en/imis?module=dataset&dasid=5188"
dasid <- fromJSON(paste0(url, "&show=json"))

# Build citation. Get active DOI.
doi <- dasid[["dois"]]
doi <- subset(doi, doi$CurrentDOI == 1)$DOI
doi <- paste0("https://doi.org/", doi)
citation <- dasid[["datasetrec"]][["Citation"]]
citation <- paste0("> ", citation, " ", doi, 
                   ". Accessed through the LifeWatch Data Explorer / lwdataexplorer R package."
                   )

# Get license
license <- dasid[["datasetrec"]][["AccConstrDisplay"]]
```

```{r display citation, results = 'asis', echo = FALSE}
cat(citation)
```

```{r display license, results = 'asis', echo = FALSE}
cat(paste0("<b>Availability:</b>", license))
```

Please acknowledge as: This work makes use of the LifeWatch observation data and infrastructure (provided by VLIZ) funded by Research Foundation - Flanders (FWO) as part of the Belgian contribution to LifeWatch.

These data are also available in the R language with the [lwdataxplorer package](https://lifewatch.github.io/lwdataexplorer/).
